src/middleware/log_request.rs

```diff
@@ -186,8 +152,10 @@ impl Display for RequestLine<'_> {
             line.add_quoted_field("error", err)?;
         }
 
-        if self.response_time > SLOW_REQUEST_THRESHOLD_MS {
-            line.add_marker("SLOW REQUEST")?;
+        if let Some(response_time) = response_time {
+            if response_time.as_millis() > SLOW_REQUEST_THRESHOLD_MS {
+                line.add_marker("SLOW REQUEST")?;
+            }
```

Comment:
We could de-duplicate this `if let` by merging it into the above `if let` or binding `response_time`.

Reply:
the problem with merging them is that it changes the field order

Reply:
Ah, I see, then it should be fine. Thanks for clarifying!